### DISCUSSION TITLE
请问中文的语言包在哪里

### DISCUSSION THREAD

**Discussion by jefferyjob** [element-plus/element-plus#15093] [Q&A]
https://github.com/element-plus/element-plus/tree/dev/docs/en-US 

整个里面为什么只有英文的语言包，没有中文的语言包。我看官网是有中文语言包的，请问一下其他语言的语言包在哪里呢？

**Comment by xing403** [comment]
[element-plus github 文档仓库 gh-pages 分支](https://github.com/element-plus/element-plus/tree/gh-pages)
[element-plus gitee 文档仓库  gh-pages 分支](https://gitee.com/element-plus/element-plus/tree/gh-pages)
*gitee 仓库的代码应该是从 github 仓库同步过去的*

[element-plus 不同语言关于文档翻译的介绍](https://element-plus.org/zh-CN/guide/translation.html)

**Comment by rzzf** [comment]
See https://github.com/element-plus/element-plus/discussions/24694
